assets/csi_cloud_config.ini

```diff
@@ -2,6 +2,8 @@
 
 [Global]
 cluster-id = "${CLUSTER_ID}"
+user = "${USER}"
+password = "${PASSWORD}"
```

Comment:
Do we need to escape weird characters in the password? I assume env. var could have anything except '\0' and we did not need to care about escaping there. What are rules in the ini file?

Reply:
Escaping backslashes and quotation marks.